website/content/en/docs/upgrading/v1-migration.md

```diff
@@ -39,63 +153,31 @@ The upgrade guide will first require upgrading to your latest patch version prio
     export KARPENTER_IAM_ROLE_ARN="arn:${AWS_PARTITION}:iam::${AWS_ACCOUNT_ID}:role/${CLUSTER_NAME}-karpenter"
     ```
 
-
-2. Determine the current Karpenter version:
-   ```bash
-   kubectl get pod -A | grep karpenter
-   kubectl describe pod -n "${KARPENTER_NAMESPACE}" karpenter-xxxxxxxxxx-xxxxx | grep Image:
-   ```
-   Sample output:
+2. Determine your current Karpenter version:
    ```bash
-   Image: public.ecr.aws/karpenter/controller:0.37.1@sha256:157f478f5db1fe999f5e2d27badcc742bf51cc470508b3cebe78224d0947674f
+   kubectl get pod -A -l app.kubernetes.io/name=karpenter -ojsonpath='{.items[0].spec.containers[0].image}' | sed -e 's/^[^:]*:\([^@]*\).*$/\1/'
```

Comment:
I get a trailing % like `v0.36.0%` when using this regex, should we instead use
```
k get deployment karpenter -n kube-system -o yaml | grep app.kubernetes.io/version | awk '{print $2}'
```

Reply:
The percent sign is from your shell, not the command. It just means there was no new line character. The version label is definitely a better source than the image though, I'll replace with this:
```bash
kg deployment -A -l app.kubernetes.io/name=karpenter -ojsonpath="{.items[0].metadata.labels['app\.kubernetes\.io/version']}{'\n'}"
```